ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml

```diff
@@ -83,6 +83,12 @@ Additional_Build_Tools_RHEL_x86:
   - libstdc++.i686                # a dependency required for executing a 32-bit C binary
   - libnss3.so
 
+Additional_Build_Tools_RHEL10_x86:
```

Comment:
A couple of things:
1. The glibc addition was intended to be specifically to allow th installation of the 32-bit (i686) packages to allow building the JDK for x86-32. If we can't install those on RHEL10 then I'd just skip them completely.
2. nss /should/ work in place of `libnss3.so` so we can possibly just use that in the pre-10 definition here.
3. If we still need to the RHEL10 can we make it `Additional_Build_Tools_RHEL10Plus_x86` so that it doesn't cause confusion when we get a RHEL11?

Reply:
Resolved, leaving 32bit packages for pre-rhel10, and just adding nss for rhel10